### ISSUE TITLE
BoxEdit has no data (data is always None)

### ISSUE THREAD

**Issue by N4321D** [holoviz/holoviews#5471]
#### ALL software version info
holoviews: 1.15.0
bokeh: 2.4.3
jupyterlab: 3.4.4
python (anaconda): 3.9.13
OS: Linux Mint 21 Cinnamon
browser: Chromium

#### Description of expected behavior and the observed behavior
When running the example on https://holoviews.org/reference/streams/bokeh/BoxEdit.html
the box_stream.data is always None

#### Complete, minimal, self-contained example code that reproduces the issue
MRE (from the example on holoviews.org):

```
import holoviews as hv
from holoviews import opts
from holoviews import streams
hv.extension('bokeh')

boxes = hv.Rectangles([(0, 0, 1, 1), (2, 1, 3.5, 2.5), (0.5, 1.5, 1.5, 2.5)])
box_stream = streams.BoxEdit(source=boxes, num_objects=3, styles={'fill_color': ['red', 'green', 'blue']})
boxes.opts(
    opts.Rectangles(active_tools=['box_edit'], fill_alpha=0.5, height=400, width=400))

print(box_stream.data)
```
output:
```None```

![image](https://user-images.githubusercontent.com/35295509/194174469-cd0a1bb1-5869-4cd8-a0dd-154e7eb624e6.png)

**Comment by remborg** [comment]
This is quite old but have you managed to find a fix for this?

I've got the same issue with:

> holoviews: 1.15.3
> bokeh: 2.4.3
> jupyterlab: 3.6.8
> python: 3.9.12

**Comment by hoxbro** [comment]
The versions reported here are pretty old; running newer versions still has that problem. It looks like the initial state isn't synced, but it is once we move one of the rectangles.

**Comment by remborg** [comment]
Thanks for your help @hoxbro 
I also have the problem after drawing a rectangle, `box_stream.data` stays None.

**Comment by hoxbro** [comment]
It works for me, can you try to update to newer versions? The latest holoviews version is 1.22.1, and Bokeh is 3.8.2.

**Comment by remborg** [comment]
Unfortunately I'm stuck with these versions for now because of dependencies...
However I see that I'm missing pyviz_comm. Could that be the reason? 
I'll try to install it and see if it works.

**Comment by hoxbro** [comment]
`pyviz_comms` is definitely needed in both the environment you are running from and the one where JupyterLab is launched (could be the same).

**Comment by remborg** [comment]
Indeed, installing pyviz_comms 2.2.0 fixed my issue. Thanks for your help.
Should this be added to the dependencies list?

**Comment by hoxbro** [comment]
It already is (atleast on newer versions):

https://github.com/holoviz/holoviews/blob/743136dd5cd148e79fe5b5c7225a55d47babfd09/pyproject.toml#L43
